CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to AdGuard filters
+
+Short introduction, explain what the ways to contribute are.
+
+## How to work with this repo
+
+1. Install [Visual Studio Code](https://code.visualstudio.com/download) with the ["Adblock" syntax plugin](https://marketplace.visualstudio.com/items?itemName=adguard.adblock).
+2. Clone (or fork and clone) [this repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository#cloning-a-repository).
+3. Make your changes and commit them.
+4. Every commit needs to be linked to an issue. The commit message should look like (`Fix #123` where `123` is the issue number). This way the issue will be closed automatically once this commit lands in the `master` branch.
+
+## How to write filter rules
+
+Before you start, please also note we follow to the current [filters policy](https://kb.adguard.com/en/general/adguard-filter-policy). You should read and understand it - one of the most important things is the [quality requirements](https://kb.adguard.com/en/general/adguard-filter-policy#quality-requirements-for-filtering-rules).
```

Comment:
Remove `/en/` from links. The site redirects to correct lang.

Reply:
Removed it.